### PR TITLE
Enhance output of 'stratis fs list'

### PR DESCRIPTION
**Pull request by agrover**
* Based on example of previous work on blockdevs, do not require pool name to be given.
* Add output of newly added fields: Created date, used space, and mount points.
* Also print Devnode, which is now `/dev/stratis/<poolname>/<fsname>`.

Requires additional dependency on python-dateutil.

### CODE REVIEW

**Comment by agrover** [comment]
Example output of `stratis fs list`:
```
Pool Name  Name  Used        Created            Device                  Mount Points  
test1      fs1   > 1.05 GiB  Aug 10 2018 12:41  /dev/stratis/test1/fs1  /root/mnt, /root/mnt2
```

**Comment by mulkieran** [comment]
> Upon reflection, I think maybe we don't want to list mount points. One, because it's potentially a list, which is hard to show in fs list, and also because this information is already available with the mount command that users are already likely familiar with. Thoughts?

Less is more, AFAIC.

**Comment by agrover** [comment]
> Also, please run up some tests as well. test/integration/logical/test_list.py is probably the correct place.

Any ideas for good things to test here?

**Comment by agrover** [comment]
OK, added stuff to test, as much as we can.

File: src/stratis_cli/_actions/_logical.py [hunk 1]
```diff
@@ -66,20 +70,26 @@ def list_volumes(namespace):
...
+            MOFilesystem(info) for _, info in filesystems(props=properties)
+            .search(managed_objects)
+        ]
 
-        print_table(['Name'], sorted(tables, key=lambda entry: entry[0]),
-                    ['<'])
+        tables = [[
+            path_to_name[mofilesystem.Pool()],
+            mofilesystem.Name(),
+            str(Range(mofilesystem.Used(), 1)),
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_logical.py]: ```str(Range(mofilesystem.Used()))``` is better. I think the "1" can not do any harm, but it should be entirely unnecessary.

File: src/stratis_cli/_parser/_parser.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 import argparse
 
 from .._actions import PhysicalActions
+from .._actions import LogicalActions
```
Comment (mulkieran) [inline src/stratis_cli/_parser/_parser.py]: Please reverse order, "L" before "P".

File: tests/integration/logical/test_list.py [hunk 3]
```diff
@@ -123,3 +123,53 @@ def testList(self):
...
+                        ] + self._VOLUMES[2]
+        RUNNER(command_line)
+
+    def tearDown(self):
+        """
+        Stop the stratisd simulator and daemon.
+        """
+        self._service.tearDown()
+
+    def testList(self):
```
Comment (mulkieran) [inline tests/integration/logical/test_list.py]: If you are only going to use ```_MENU``` in one place you might as well get rid of it entirely and just write it out explicitly in first line.

**Comment by agrover** [comment]
cool, thanks. will do.

**Pull request closed without merging**